.github/workflows/shared_workflow.yml

```
@@ -11,15 +11,17 @@ jobs:
     name: unit tests and linting
     strategy:
       matrix:
-        nextcloudVersion: [ stable26, stable27, stable28, stable29, stable30 ]
-        phpVersion: [ 8.0, 8.1, 8.2, 8.3]
-        exclude:
+        nextcloudVersion: [ stable30 ]
```

```suggestion
        nextcloudVersion: [ stable30 ]
```
Since this app supports Nextcloud versions 26 to 30, we need to ensure that our code is maintained accordingly. 
https://github.com/nextcloud/integration_openproject/blob/master/appinfo/info.xml#L44.